tests/src/JIT/SIMD/Vector3TestNative.cpp

```diff
@@ -21,6 +21,10 @@
 #endif  // !__i386__
 #endif // !defined(_MSC_VER)
 
+#ifndef _countof
```

Comment:
All of this stuff belongs in a common header, but we currently don't have that.  The Interop tests have some common stuff that should probably be moved to a common location that all tests can share.